src/components/header/Header.css

```diff
@@ -1,7 +1,81 @@
 .header {
+	margin: -10px 0 0 -10px;
+	width: 100%;
+}
+
+.p-header {
 	display: flex;
 	justify-content: space-between;
 	align-items: center;
 	background-color: #f0f0f0;
 	padding: 10px 20px;
 }
+
+.hero {
+	width: 100vw;
+}
+
+.hero img {
+	width: 100vw;
+	height: auto;
+}
+
+h1 {
+	position: absolute;
+	top: 200px;
+	font-size: 70px;
+	color: white;
+	left: 271px;
+}
+
+.heroBookmark {
+	background-color: yellow;
+	padding: 5px 80px 3px 40px;
+	font-size: 38px;
+	position: absolute;
+	width: 200px;
+	height: 49px;
+	top: 481px;
+	left: 271px;
+	display: flex;
+	gap: 15%;
+	align-items: center;
+	color: black;
+	border-radius: 60px;
+}
+
+.hero-text {
+	position: absolute;
+	font-family: Content;
```

Comment:
Nincs ilyen beépített font family, de ez kb 95% arial, amit be is állítottam az app.css-ben.